attending/index.py

```diff
@@ -0,0 +1,19 @@
+import csv
+from pathlib import Path
+import os
+
+
+def load_index(file: Path):
+    project_index = {}
+    assert file.is_file()
```

Comment:
```suggestion
```

Also, assertions get eliminate when running python with optimization turned on.

```
$ python -c "assert True == False, 'oops'; print('ok')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AssertionError: oops
$ python -O -c "assert True == False, 'oops'; print('ok')"
ok
```

It's no better to fail here compared to failing on the file `open()` call, but leaving this out allows us to accept strings and bytes as the input parameter.